shared/common-adapters/avatar.render.native.tsx

```diff
@@ -55,14 +55,15 @@ const Avatar = (props: Props) => {
             ]}
           />
         )}
-        {(!!props.borderColor || props.isTeam) && (
-          <Kb.Box
-            style={[
-              styles.borderBase,
-              {borderColor: props.borderColor || Styles.globalColors.black_10, borderRadius},
-            ]}
-          />
-        )}
+        {(!!props.borderColor || props.isTeam) &&
+          false && ( // looks better off i think
```

Comment:
should we turn off on desktop too? why `false &&` instead of just ripping it out?

Reply:
cause i wanted to see how the team reacted